package.json

```diff
@@ -4,11 +4,11 @@
   "private": true,
   "dependencies": {
     "@patternfly/patternfly": "4.70.2",
-    "@patternfly/react-console": "2.0.16",
+    "@patternfly/react-console": "4.1.25",
     "@patternfly/react-core": "4.84.4",
     "@patternfly/react-table": "4.19.24",
     "@redhat/redhat-font": "git+https://github.com/RedHatOfficial/RedHatFont.git#2.2.0",
-    "@novnc/novnc": "1.1.0",
+    "@novnc/novnc": "1.2.0",
```

Comment:
This should be dropped entirely again. Commit 76f48027 introduced it as a hack, but I think it's better to leave control of the novnc version to react-console.

Reply:
I had some problems with removing it, with unatisfied deps, I can try once again. Maybe after mergin this?

Reply:
Not important at all, fine for me to merge as-is (and I approved). Not sure about the issue @skobyda just found, if it's just harmless cleanup or a blocker.